util.go

```diff
@@ -94,16 +94,33 @@ func insensitiviseMap(m map[string]interface{}) {
 	}
 }
 
+func getEnv(key string) string {
+	if key == "HOME" && runtime.GOOS == "windows" {
+		home := os.Getenv(key)
+
+		if home != "" {
+			return home
+		}
+
+		home = os.Getenv("HOMEDRIVE") + os.Getenv("HOMEPATH")
+
+		if home != "" {
+			return home
+		}
+
+		return os.Getenv("USERPROFILE")
+	}
+
+	return os.Getenv(key)
+}
+
 func absPathify(inPath string) string {
 	jww.INFO.Println("Trying to resolve absolute path to", inPath)
 
-	if strings.HasPrefix(inPath, "$HOME") {
-		inPath = userHomeDir() + inPath[5:]
-	}
+	inPath = filepath.FromSlash(inPath)
 
-	if strings.HasPrefix(inPath, "$") {
-		end := strings.Index(inPath, string(os.PathSeparator))
-		inPath = os.Getenv(inPath[1:end]) + inPath[end:]
+	if strings.ContainsRune(inPath, '$') {
```

Comment:
This will fail builds for the same reason mentioned [here](https://github.com/spf13/viper/pull/287#issuecomment-267423883). The solution is simple, as mentioned [here](https://github.com/spf13/viper/pull/287#issuecomment-267463231).